### DISCUSSION TITLE
Is there an simple Search Feed from Video Feed generator?

### DISCUSSION THREAD

**Discussion by jsonnentag** [rrirower/DP2SceneGraphWizard#101] [General]
I finally realized that just because you did the Deep Search Validation thing doesn't mean anything is searchable and visible to anybody.  :)

Does anybody know of anything that takes contents of a video feed and creates a search feed from it?  There would have to be a few extra things (it seems like most can be default settings for language and country availability), but most other stuff seems to be in the video feed.

Roku's examples: [https://github.com/rokudev/search-feed-json](url)

Their self-serve feed submission location: [https://developer.roku.com/apps/search/overview](url)

**Comment by jsonnentag** [comment]
I guess it always helps to look at the abilities of the Feed Manager that exists.  :)

[https://github.com/rrirower/Channel-Feed-Manager](url)

Channel Feed Manager

The Channel Feed Manager is a software suite of tools that assist in creating and managing feed files for Roku and Amazon Creator. Using these tools will allow you to spend less time worrying about the syntactical correctness of your feed file and more time developing your video content.

NOTE: This software is not endorsed by or affiliated with Roku or Amazon in any way.
Features

    Manage Roku Json and Xml feed files.
    Manage Amazon Creator feed files.
    Includes FeedMerge tool to merge one or more Json feed files.
    Includes TrickplayMaker to generate BIF files.
---> Generates Search feeds for your feed file.
    Supports all Roku feed content types.
    Import/Export to other feed types.
    Feed file validation.

**Comment by jsonnentag** [comment]
Oh boy!  I got into a massive never ending error loop when I imported my feed and just tried exporting a search feed from it.  :))

![image](https://github.com/rrirower/DP2SceneGraphWizard/assets/36777806/55d88a10-f2fb-4cb2-b85d-a9fd6e564713)

**Comment by rrirower** [comment]
This should be resolved.
